### PR TITLE
update upgrade case

### PR DESCRIPTION
**Pull request by lihongyan1**
Jenkins job
https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/ocp-common/job/Runner/680401/

### CODE REVIEW

**Review by geliu2016 — APPROVED** [review]
/lgtm

File: features/upgrade/monitoring/monitoring-upgrade.feature [hunk 1]
```diff
@@ -76,3 +76,16 @@ Feature: cluster monitoring related upgrade check
     When I run the :oadm_top_node admin command
     Then the output should contain:
       | CPU(cores) |
+
+    # curl -k -H "Authorization: Bearer $token" 'https://prometheus-k8s.openshift-monitoring.svc:9091/api/v1/query?query=cluster_installer'
```
Comment (juzhao) [inline features/upgrade/monitoring/monitoring-upgrade.feature:80]: prometheus-k8s
->
thanos-querier

**Pull request merged**